acceptance/bundle/git-permerror/output.txt

```diff
@@ -3,6 +3,7 @@
 >>> chmod 000 .git
 
 >>> $CLI bundle validate
+Warn: failed to read .git: unable to load repository specific gitconfig: open config: permission denied
 Error: unable to load repository specific gitconfig: open config: permission denied
```

Comment:
If I'm correct, the warning ends up on stderr and the error on stdout. If so, could be problematic.

Reply:
I agree it's odd but it's backward compatible.

In this particular case the same issue is reported both as warning and as error - that part needs clean up imo. Then issue will disappear.